autoware_launch/launch/components/tier4_planning_component.launch.xml

```diff
@@ -45,21 +45,22 @@
-    <arg name="stop_line_param_path" value="$(var behavior_velocity_config_path)/stop_line.param.yaml"/>
-    <arg name="traffic_light_param_path" value="$(var behavior_velocity_config_path)/traffic_light.param.yaml"/>
-    <arg name="virtual_traffic_light_param_path" value="$(var behavior_velocity_config_path)/virtual_traffic_light.param.yaml"/>
-    <arg name="occlusion_spot_param_path" value="$(var behavior_velocity_config_path)/occlusion_spot.param.yaml"/>
-    <arg name="no_stopping_area_param_path" value="$(var behavior_velocity_config_path)/no_stopping_area.param.yaml"/>
-    <arg name="run_out_param_path" value="$(var behavior_velocity_config_path)/run_out.param.yaml"/>
-    <arg name="speed_bump_param_path" value="$(var behavior_velocity_config_path)/speed_bump.param.yaml"/>
-    <arg name="out_of_lane_param_path" value="$(var behavior_velocity_config_path)/out_of_lane.param.yaml"/>
-    <arg name="no_drivable_lane_param_path" value="$(var behavior_velocity_config_path)/no_drivable_lane.param.yaml"/>
-    <arg name="behavior_velocity_planner_param_path" value="$(var behavior_velocity_config_path)/behavior_velocity_planner.param.yaml"/>
+    <arg name="behavior_velocity_smoother_type_param_path" value="$(find-pkg-share autoware_launch)/config/planning/scenario_planning/common/motion_velocity_smoother/Analytical.param.yaml"/>
+    <arg
+      name="behavior_velocity_planner_common_param_path"
+      value="$(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/behavior_velocity_planner.param.yaml"
+    />
+    <arg
+      name="behavior_velocity_planner_module_param_paths"
+      value="[$(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/blind_spot.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/crosswalk.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/detection_area.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/intersection.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/stop_line.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/traffic_light.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/no_stopping_area.param.yaml,
+              $(find-pkg-share autoware_launch)/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_velocity_planner/out_of_lane.param.yaml]"
```

Comment:
How about using ```behavior_velocity_config_path``` to omit the description?

Reply:
Updated https://github.com/autowarefoundation/autoware_launch/pull/369/commits/441fe2f3ad4884cb582d1e7f39a2b1bf64dbf6ec.